internal/util/context.go

```diff
@@ -19,8 +19,15 @@ func ContextWithLogger(ctx context.Context, logger *logrus.Entry) context.Contex
 
 // LoggerFromContext returns the logger from the context.
 func LoggerFromContext(ctx context.Context) *logrus.Entry {
```

Comment:
in the long run we can rename this to `LoggerFrom` (as it's clear it's from  ctx).

open an issue to track pls.